docs/team/yungweezy.md

```diff
@@ -46,17 +46,20 @@ Given below is a summary of my contributions to the development of TutorBuddy.
 ### Other Contribution
```

Comment:
Haha anw for the PPP, there is a hard limit of 2 page after converting it to PDF. I decided to remove the justification part for the Major enhancement, if I am not wrong, yours also exceeds, might want to remove some stuff here as well to try to fit to 2 pages ah.

Reply:
God it thank you!